packages/use-contractkit/src/contract-kit-provider.tsx

```diff
@@ -97,13 +97,16 @@ export const ContractKitProvider: React.FC<ContractKitProviderProps> = ({
   feeCurrency = CeloContract.GoldToken,
 }: ContractKitProviderProps) => {
   const isMountedRef = useIsMounted();
-  const previousConfig = useMemo(() => loadPreviousConfig(network), [network]);
+  const previousConfig = useMemo(
+    () => loadPreviousConfig(network, feeCurrency),
+    [network]
```

Comment:
missing feeCurrency

Reply:
Actually in this case I would argue it's the reverse issue. We shouldn't have network in there because we never want to pull the localstorage data after mounting. The real purpose of the `useMemo` is to prevent unnecessarily calling `loadPreviousConfig` and if feeCurrency or network changed for some reason in the props we don't really care for the initial reducer state because it was already set.